### ISSUE TITLE
Chat history persistence

### ISSUE THREAD

**Issue by jmcbee** [erming/shout#309]
At the moment, cache's stored into a [variable](https://github.com/erming/shout/blob/master/src/models/chan.js#L16).

What do you think? :stuck_out_tongue_winking_eye:

**Comment by nnevala** [comment]
I think this is currently the biggest missing feature from Shout. Ideally the history would be persistent, loaded up on server restarts, searchable, etc...

**Comment by davibe** [comment]
+1

**Comment by furier** [comment]
yeah the logs are still there after a restart of the shout server but they are not loaded into the client which is a shame, it should be a very easy thing to do.

**Comment by nickel715** [comment]
+1

**Comment by floogulinc** [comment]
See PR #391
